app/src/main/java/space/taran/arknavigator/mvp/model/repo/stats/PlainStatsStorage.kt

```diff
@@ -45,6 +55,11 @@ class PlainStatsStorage(
     }
 
     override fun handleEvent(event: StatsEvent) {
+        if (!collectingTagEvents &&
+            StatsStorage.TAGS_USAGE_EVENTS.contains(event.javaClass)
```

Comment:
Can't we destructure `event` by cases in Kotlin without using Java Reflection API?

Reply:
Actually, there is no reflection here, class of object is available by default and does not create overhead.